src/sentry/notifications/notifications/activity/base.py

```diff
@@ -129,12 +130,25 @@ def get_context(self, provider: ExternalProviders | None = None) -> MutableMappi
         should_add_url = provider is not None
         text_description = self.description_as_text(text_template, params, should_add_url, provider)
         html_description = self.description_as_html(html_template or text_template, params)
-        return {
+
+        enhanced_privacy = self.group.organization.flags.enhanced_privacy
+
+        context = {
             **self.get_base_context(),
             "text_description": text_description,
             "html_description": html_description,
+            "enhanced_privacy": enhanced_privacy,
         }
 
+        if features.has("organizations:suspect-commits-in-emails", self.group.organization):
+            # Add suspect commits to workflow notifications using the same method as issue alerts
+            if self.group:
```

Comment:
would prefer to combine the two checks into one statement, to prevent [code arrowing](https://blog.codinghorror.com/flattening-arrow-code/)